lib/src/main/java/com/auth0/msg/ClaimType.java

```diff
@@ -0,0 +1,18 @@
+package com.auth0.msg;
+
+import java.util.Arrays;
+import java.util.List;
+
```

Comment:
Please add Javadocs. As we discussed previously - we shoould look at how we will use this enum to capture the type of the claim. I am talking about Claim type that you have mentioned here https://docs.google.com/document/d/1-N0n7UopFaIhzA5X-j1fhBgAR-kImbKoqgSVTUmixEI/edit?ts=5aac5b78#bookmark=id.u4q83iah8gx9

If we need to validate the type, we should have type information in the enum itself.